Tasks/AzureRmWebAppDeployment/KuduUtility.ts

```diff
@@ -0,0 +1,89 @@
+import Q = require('q');
+import tl = require('vsts-task-lib/task');
+import path = require("path");
+import fs = require("fs");
+var request = require ('request');
```

Comment:
Ensure you handle a private agent connecting to VSTS from behind a proxy.  Nuget team just had multiple LSIs after they released their task and had to revert some of the task.